src/components/Proposal/Create/CreateProposal.scss

```diff
@@ -388,7 +387,7 @@
   content: "";
   clear: both;
   overflow: auto;
-  height: calc(100% - 40px);
+  max-height: calc(100vh - 200px);
```

Comment:
how does 200px look on mobile?  Would a % be better?

Reply:
I agree that generally `%` is preferable but in this case, probably because the way it's built, it's not look like expected with `%`. Regarding the mobile it looks good, I've done a small modification
now.

Reply:
> it's not look like expected with %

I don't understand what you're saying

Reply:
It just does not works good with `%`, that's the reason I use `vh` there.